crates/aptos/README.md

```diff
@@ -1,4 +1,4 @@
 # Aptos Command Line Interface (CLI) Tool
 
 The `aptos` tool is a command line interface (CLI) for debugging, development, and node operation.
-See [Aptos CLI Documentation](https://aptos.dev/cli-tools/aptos-cli-tool/install-aptos-cli) for how to install the `aptos` CLI tool and how to use it.
+See [Aptos CLI Documentation](https://aptos.dev/cli-tools/aptos-cli/use-cli/install-aptos-cli) for how to install the `aptos` CLI tool and how to use it.
```

Comment:
Remember every time you change a link like this, you could break something like google links by shortening aptos-cli-tool to aptos-cli.

I think there's a way to do links

Reply:
Looks the previous link was broken already lol. It should be - https://aptos.dev/tools/install-cli/

I will update my changes as well